### PR TITLE
Remove ref to franklindocs repo

### PR DESCRIPTION
**Pull request by remi-garcia**
I found three links to an archived repo.

I am not sure about the change in the README.md (source -> see docs/). What do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Go to [Franklin's main website](https://franklinjl.org).
 
 Some examples of websites using Franklin (_if you're using Franklin with a public repo, consider adding the "franklin" tag to the repo to help others find examples, thanks!_)
 
-* Franklin's own website is written in Franklin, [source](https://github.com/tlienart/franklindocs)
+* Franklin's own website is written in Franklin, [see docs/](docs/)
```
Comment (tlienart) [inline README.md:38]: I don't think that works, https://github.com/tlienart/Franklin.jl/tree/master/docs is probably safer

**Review by tlienart — CHANGES_REQUESTED** [review]

**Comment by tlienart** [comment]
great, thanks!

File: README.md (hunk 1 above)
Comment (fredrikekre) [reply README.md:38]: I think it works, and has the additional benefit that it links to the correct commit, but perhaps you always want people to see the `master` branch?

Comment (tlienart) [reply README.md:38]: no that's great, I learned something!

**Pull request merged**